tests/e2escenarios/e2e_images_test.go

```diff
@@ -111,11 +99,6 @@ var _ = Describe("odo supported images e2e tests", func() {
 			verifySupportedImage("redhat-openjdk-18/openjdk18-openshift:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
-		It("Should be able to verify the openjdk-11-rhel7 image", func() {
```

Comment:
why do you want to remove this?

Reply:
because there is a openjdk-11 already present

Reply:
added them back

Reply:
or we run them in the periodic job or we fix them if someone complains about them.

Reply:
> because there is a openjdk-11 already present

but that is a different image :-/ I know that this is a mess with all the different images. But this is why we need those tests because the images are build differently and we need to verify that odo works with them.

Reply:
I added the images back. Removed a rhel-7 one as there is rhel-8 for same java version

Reply:
> I added the images back. ~~ Removed a rhel-7 one as there is rhel-8 for same java version~~
> All images were added back

Please read carefully @prietyc123

Reply:
Typo from my end that didn't render the markdown as cancelled line, I apologize.